### DISCUSSION TITLE
Tecnologia x Impacto social

### DISCUSSION THREAD

**Discussion by dionarodrigues** [frontendbr/forum#69] [Discussão]
Google, Samsung, dentre tantas outras empresas vêm estimulando a criação de produtos tecnológicos que causam um impacto social, mudando a sociedade para o melhor. 

O braço filantrópico do Google lançou um desafio para ONGs brasileiras, onde 10 milhões de reais serão distribuídos em forma de investimento para as melhores ideias que utilizarem a tecnologia para solucionar alguma questão social.  Veja: https://desafiosocial.withgoogle.com/brazil2016

---

No primeiro e no segundo semestre do curso de Produção Multimídia, no Centro Universitário do SENAC, minha turma se dividiu em grupos para criar um projeto e, consequentemente, um produto 100% digital, que causasse impacto social. Além disso, o produto precisaria ser sustentável. Foi genial! Aprendi muitas coisas, tive muitas ideias e com certeza expandi muito a  minha visão sobre esse assunto e sua importância para o mundo.

---

Como você vê essas ações? Conhece outras iniciativas? Já participou de algum projeto do gênero?

**Comment by bernardodiasc** [comment]
Aqui em Ubatuba ta rolando um movimento interessantíssimo nesse aspecto, temos o apoio e vontade pública do nosso lado (comunidade) e ta ganhando bastante tração. Saiu uma materia sobre aqui http://www.informarubatuba.com/#!simciti/ccqk, o link da minuta para consulta pública está no final da matéria, recomendo a leitura e quem sabe outras cidades podem replicar a ideia.
